source/agora/test/ValidatorCleanRestart.d

```diff
@@ -93,16 +89,16 @@ version(none) unittest
     // Check if the validators in the set B have all the addresses for
     // current validators and previous validators except themselves.
```

Comment:
```suggestion
    // current validators and previous validators.
```
since we're checking `>=` now

Reply:
I will update the comment and the check to be an exact count once I double check what it should be.

Reply:
```
["http://validator-0.boa1xrval5rzmma29zh4aqgv3mvcarhwa0w8rgthy3l9vaj3fywf9894ycmjkm8.localrest/", 
"http://validator-3.boa1xrval6hd8szdektyz69fnqjwqfejhu4rvrpwlahh9rhaazzpvs5g6lh34l5.localrest/", 
"http://validator-2.boa1xrval7gwhjz4k9raqukcnv2n4rl4fxt74m2y9eay6l5mqdf4gntnzhhscrh.localrest/", 
"http://validator-1.boa1xzval3ah8z7ewhuzx6mywveyr79f24w49rdypwgurhjkr8z2ke2mycftv9n.localrest/", 
"http://validator-6.boa1xza007gllhzdawnr727hds36guc0frnjsqscgf4k08zqesapcg3uujh9g93.localrest/", 
"http://validator-5.boa1xzval2a3cdxv28n6slr62wlczslk3juvk7cu05qt3z55ty2rlfqfc6egsh2.localrest/", 
"http://validator-8.boa1xqd00qsu7n5ykyckc23wmcjglfalcdea3x2af88hx2x5qx65x7w8g2r5t29.localrest/", 
"http://name.registry/", 
"http://validator-4.boa1xzval4nvru2ej9m0rptq7hatukkavemryvct4f8smyy3ky9ct5u0s8w6gfy.localrest/", 
"http://validator-7.boa1xrc00kar2yqa3jzve9cm4cvuaa8duazkuwrygmqgpcuf0gqww8ye7ua9lkl.localrest/"]
```
It should include the registry and all validators.